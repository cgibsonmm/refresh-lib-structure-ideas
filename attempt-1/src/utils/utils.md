# Utils

This folder will house global used Utils

I.E.

1. commaFormat
