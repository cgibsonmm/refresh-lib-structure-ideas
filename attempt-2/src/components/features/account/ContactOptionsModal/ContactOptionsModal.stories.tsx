import { ContactOptionsModal } from './ContactOptionsModal';
import type { Meta, StoryObj } from '@storybook/react';

const meta: Meta<typeof ContactOptionsModal> = {
  title: 'Components/ContactOptionsModal',
  component: ContactOptionsModal,
  parameters: {
    docs: {
      story: {
        inline: false,
        iframeHeight: 600,
      },
    },
  },
  argTypes: {
    setIsOpen: { action: 'setIsOpen' },
    onCloseHandle: { action: 'onCloseHandle' },
    openConfirmCancelModal: { action: 'openConfirmCancelModal' },
  },
  args: {
    isOpen: true,
  },
};

export default meta;

type Story = StoryObj<typeof ContactOptionsModal>;

export const Default: Story = {};
