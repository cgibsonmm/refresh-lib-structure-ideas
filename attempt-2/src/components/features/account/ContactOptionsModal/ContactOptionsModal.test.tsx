import { render } from '@testing-library/react';
import { withProviders } from '@/utils/TestUtils';
import { ContactOptionsModal } from './ContactOptionsModal';

import 'jest-styled-components';

describe('ContactOptionsModal', () => {
  it('matches the snapshot', () => {
    // Use 'baseElement' because MUI renders the Dialog in a portal
    const { baseElement } = render(
      withProviders(() => (
        <ContactOptionsModal
          isOpen={true}
          setIsOpen={() => {
            return null;
          }}
          onCloseHandle={() => {
            return null;
          }}
          openConfirmCancelModal={() => {
            return null;
          }}
        />
      ))
    );

    expect(baseElement).toMatchSnapshot();
  });
});
