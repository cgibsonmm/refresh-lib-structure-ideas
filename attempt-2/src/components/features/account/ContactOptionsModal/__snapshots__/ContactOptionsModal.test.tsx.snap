// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactOptionsModal matches the snapshot 1`] = `
<body
  style="padding-right: 1024px; overflow: hidden;"
>
  <div
    aria-hidden="true"
  />
  <div
    class="MuiDialog-root MuiModal-root css-sejokn-MuiModal-root-MuiDialog-root"
    role="presentation"
  >
    <div
      aria-hidden="true"
      class="MuiBackdrop-root MuiModal-backdrop css-yiavyu-MuiBackdrop-root-MuiDialog-backdrop"
      style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    />
    <div
      data-testid="sentinelStart"
      tabindex="0"
    />
    <div
      class="MuiDialog-container MuiDialog-scrollPaper css-hz1bth-MuiDialog-container"
      role="presentation"
      style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      tabindex="-1"
    >
      <div
        aria-labelledby=":r0:"
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation24 MuiDialog-paper MuiDialog-paperScrollPaper MuiDialog-paperWidthFalse css-4pp0nb-MuiPaper-root-MuiDialog-paper"
        role="dialog"
      >
        <button
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-18jcfqy-MuiButtonBase-root-MuiIconButton-root"
          tabindex="0"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
            data-testid="CloseIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
            />
          </svg>
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
        <h2
          class="MuiTypography-root MuiTypography-h6 MuiDialogTitle-root css-hwjpfd-MuiTypography-root-MuiDialogTitle-root"
          id="modal-title"
        >
          How can we help?
        </h2>
        <div
          class="MuiDialogContent-root css-ypiqx9-MuiDialogContent-root"
        >
          <div
            class="MuiStack-root css-f779bt-MuiStack-root"
          >
            <div
              class="MuiStack-root css-b55nah-MuiStack-root"
            >
              <div
                class="MuiFormGroup-root css-dmmspl-MuiFormGroup-root"
                role="radiogroup"
              >
                <div
                  class="MuiStack-root css-1hanliz-MuiStack-root"
                >
                  <div
                    class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-160pzxy-MuiPaper-root-MuiCard-root"
                  >
                    <div
                      class="MuiStack-root css-6fg6va-MuiStack-root"
                    >
                      <div
                        class="MuiStack-root css-1jtdgec-MuiStack-root"
                      >
                        <span
                          class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary Mui-checked MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            checked=""
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name=":r1:"
                            type="radio"
                            value="upgrade"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-11zohuh-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                        >
                          Upgrade my account
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-160pzxy-MuiPaper-root-MuiCard-root"
                  >
                    <div
                      class="MuiStack-root css-6fg6va-MuiStack-root"
                    >
                      <div
                        class="MuiStack-root css-1jtdgec-MuiStack-root"
                      >
                        <span
                          class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name=":r1:"
                            type="radio"
                            value="cancel"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                        >
                          Cancel my account
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiStack-root css-1hanliz-MuiStack-root"
                >
                  <div
                    class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-160pzxy-MuiPaper-root-MuiCard-root"
                  >
                    <div
                      class="MuiStack-root css-6fg6va-MuiStack-root"
                    >
                      <div
                        class="MuiStack-root css-1jtdgec-MuiStack-root"
                      >
                        <span
                          class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name=":r1:"
                            type="radio"
                            value="billing"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                        >
                          Billing FAQ
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-160pzxy-MuiPaper-root-MuiCard-root"
                  >
                    <div
                      class="MuiStack-root css-6fg6va-MuiStack-root"
                    >
                      <div
                        class="MuiStack-root css-1jtdgec-MuiStack-root"
                      >
                        <span
                          class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name=":r1:"
                            type="radio"
                            value="general_support"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                        >
                          General Support
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiStack-root css-1hanliz-MuiStack-root"
                >
                  <div
                    class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-160pzxy-MuiPaper-root-MuiCard-root"
                  >
                    <div
                      class="MuiStack-root css-6fg6va-MuiStack-root"
                    >
                      <div
                        class="MuiStack-root css-1jtdgec-MuiStack-root"
                      >
                        <span
                          class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name=":r1:"
                            type="radio"
                            value="partnership_inquiries"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                        >
                          Partnership Inquiries
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-160pzxy-MuiPaper-root-MuiCard-root"
                  >
                    <div
                      class="MuiStack-root css-6fg6va-MuiStack-root"
                    >
                      <div
                        class="MuiStack-root css-1jtdgec-MuiStack-root"
                      >
                        <span
                          class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name=":r1:"
                            type="radio"
                            value="technical_question"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                        >
                          Technical Questions
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiStack-root css-1hanliz-MuiStack-root"
                >
                  <div
                    class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-160pzxy-MuiPaper-root-MuiCard-root"
                  >
                    <div
                      class="MuiStack-root css-6fg6va-MuiStack-root"
                    >
                      <div
                        class="MuiStack-root css-1jtdgec-MuiStack-root"
                      >
                        <span
                          class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name=":r1:"
                            type="radio"
                            value="press"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                        >
                          Press
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-160pzxy-MuiPaper-root-MuiCard-root"
                  >
                    <div
                      class="MuiStack-root css-6fg6va-MuiStack-root"
                    >
                      <div
                        class="MuiStack-root css-1jtdgec-MuiStack-root"
                      >
                        <span
                          class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name=":r1:"
                            type="radio"
                            value="permissible_uses"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                        >
                          Is my use case allowed?
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiStack-root css-1hanliz-MuiStack-root"
                >
                  <div
                    class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-160pzxy-MuiPaper-root-MuiCard-root"
                  >
                    <div
                      class="MuiStack-root css-6fg6va-MuiStack-root"
                    >
                      <div
                        class="MuiStack-root css-1jtdgec-MuiStack-root"
                      >
                        <span
                          class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name=":r1:"
                            type="radio"
                            value="privacy_opt_out"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                        >
                          Remove my info
                        </p>
                      </div>
                    </div>
                  </div>
                  <div
                    class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-160pzxy-MuiPaper-root-MuiCard-root"
                  >
                    <div
                      class="MuiStack-root css-6fg6va-MuiStack-root"
                    >
                      <div
                        class="MuiStack-root css-1jtdgec-MuiStack-root"
                      >
                        <span
                          class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name=":r1:"
                            type="radio"
                            value="other"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                        >
                          Other
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="MuiStack-root css-1hanliz-MuiStack-root"
                >
                  <div
                    class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root css-160pzxy-MuiPaper-root-MuiCard-root"
                  >
                    <div
                      class="MuiStack-root css-6fg6va-MuiStack-root"
                    >
                      <div
                        class="MuiStack-root css-1jtdgec-MuiStack-root"
                      >
                        <span
                          class="MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root MuiRadio-root MuiRadio-colorPrimary MuiRadio-root MuiRadio-colorPrimary css-vqmohf-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name=":r1:"
                            type="radio"
                            value="custom_larger_plan"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                        >
                          Request custom/larger plan
                        </p>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="MuiStack-root css-6amu49-MuiStack-root"
            >
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-13f2ijq-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                Continue
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-1pbgvfv-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                Take Me Back
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      data-testid="sentinelEnd"
      tabindex="0"
    />
  </div>
</body>
`;
